2025-06-01 15:38:39 [main] ERROR c.c.datas.datacenter.DataCenter - Erro ao conectar com o servidor 192.168.0.4:50001: Connection refused: connect
